code/modules/awaymissions/away_props.dm

```diff
@@ -137,3 +137,16 @@
 	if(!istype(mover))
 		return
 	return isnull(mover.ckey) == reverse
+
+/obj/effect/invisible_wall // why didnt we have this already
+	name = "invisible wall"
+	desc = "You shall not pass"
+	icon = 'icons/effects/mapping_helpers.dmi'
+	icon_state = "blocker"
+	color = COLOR_BLUE_LIGHT
+	invisibility = INVISIBILITY_MAXIMUM
+	anchored = TRUE
+
+/obj/effect/invisible_wall/CanAllowThrough(mob/living/mover, border_dir)
+	..()
```

Comment:
just use density bro
also im pretty sure we have plenty of invisible wall like effects already including playeronly_barrier (with reverse to 1)

Reply:
i got annoyed seeing we had so many fake invisible wall effects with caveats so i just made a true version with no workaround jank